package.json

```diff
@@ -9,6 +9,7 @@
     "src"
   ],
   "scripts": {
+    "testA": "jest -i src/__tests__/gaussian.test.js",
```

Comment:
you could add [`.only`](https://jestjs.io/docs/en/api#testonlyname-fn-timeout) to a test to do this